shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/network/MealApi.kt

```diff
@@ -41,6 +41,10 @@ class MealApi {
     }
 
     companion object {
-        private const val MEALS_ENDPOINT = "http://localhost:3000/Meals"
+        // iOS can use this endpoint
+//        private const val MEALS_ENDPOINT = "http://localhost:3000/Meals"
+
+        // Android needs to switch to this endpoint
+        private const val MEALS_ENDPOINT = "http://10.0.2.2:3000/meals"
```

Comment:
We probably need to to a switch platform. iOS in localhost and Android in 10.0.2.2

Reply:
Agreed, for now will leave as it is to get to Demo tomorrow and we can put it on backlog if the hackathon gets extended